xptifw/unit_test/xpti_api_tests.cpp

```diff
@@ -201,29 +201,25 @@ TEST(xptiApiTest, xptiTraceEnabled) {
   EXPECT_EQ(Result, false);
 }
 
-XPTI_CALLBACK_API void trace_point_callback(uint16_t trace_type,
-                                            xpti::trace_event_data_t *parent,
-                                            xpti::trace_event_data_t *event,
-                                            uint64_t instance,
-                                            const void *user_data) {
+void trace_point_callback(uint16_t trace_type, xpti::trace_event_data_t *parent,
+                          xpti::trace_event_data_t *event, uint64_t instance,
+                          const void *user_data) {
```

Comment:
why is this declared as "const" if the intention is to update data?

Reply:
I switched to usage of global objects here. Though I don't like both, `const_cast` is probably more trouble. Removing const is not an option, because it is part of callback API.